web/models.py

```
@@ -2805,3 +2805,169 @@ class ScheduledPost(models.Model):
+    logo = models.ImageField(upload_to="chapter_logos", blank=True)
+    website = models.URLField(blank=True)
+    discord_link = models.URLField(blank=True)
+    facebook_link = models.URLField(blank=True)
+    twitter_link = models.URLField(blank=True)
+    is_active = models.BooleanField(default=True)
+    is_featured = models.BooleanField(default=False)
+    founding_date = models.DateField(auto_now_add=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    def __str__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)